src/RubyGosu.cxx

```diff
@@ -11351,6 +11418,7 @@ _wrap_translate(int argc, VALUE *argv, VALUE self) {
 
 /* -------- TYPE CONVERSION AND EQUIVALENCE RULES (BEGIN) -------- */
 
+static swig_type_info _swigt__p_Gosu__Bitmap = {"_p_Gosu__Bitmap", "Gosu::Bitmap *", 0, 0, (void*)0, 0};
```

Comment:
If SWIG wants to add the Bitmap type, you might have forgotten to %ignore Window::to_bitmap somewhere - Bitmap should be invisible to Ruby

Reply:
Yes - missed that ;-)